commit e62eccf025aa60ec268787d2aa4a46310ed1cd60
Author: David Sommerseth <davids@openvpn.net>
Date:   Tue Dec 6 23:10:51 2016 +0100

    Fix wrong configure.ac parsing of --enable-async-push
    
    AC_ARG_ENABLE() was used wrong, which led enable_async_push to
    always be set, regardless if --enable-async-push or --disable-async-push
    was used.
    
    Also spotted the exact same patch when writing this commit message as
    GitHub PR#70.
    
    Trac: #786
    Signed-off-by: David Sommerseth <davids@openvpn.net>
    Acked-by: Lev Stipakov <lstipakov@gmail.com>
    Message-Id: <1481062251-18349-1-git-send-email-davids@openvpn.net>
    URL: https://www.mail-archive.com/openvpn-devel@lists.sourceforge.net/msg13411.html

diff --git a/configure.ac b/configure.ac
index f4073d0..27bdcc3 100644
--- a/configure.ac
+++ b/configure.ac
@@ -267,7 +267,7 @@ AC_ARG_ENABLE(
 AC_ARG_ENABLE(
 	[async-push],
 	[AS_HELP_STRING([--enable-async-push], [enable async-push support @<:@default=no@:>@])],
-	[enable_async_push="yes"],
+	,
 	[enable_async_push="no"]
 )
 
